CONTRIBUTING.md

```diff
@@ -33,7 +34,34 @@ The following text equips you with knowledge that makes contributing to ilastik
 
    ```
    conda config --set channel_priority strict
-   conda install --name base -c conda-forge mamba
+   conda install mamba -n base -c conda-forge
```

Comment:
In general this guide is thought to be a set of copy-paste operations.

not sure why short options would be better than long options. "Experts" already know them both and can probably replace while typing, while novice users profit from the added verbosity of long options.

Reply:
Didn't think about that.
Yet, the usage of both short `-c` and long option `--name` also is inconsistent in that regard.

So maybe `conda install --name base --channel conda-forge` mamba would be best suited here?

Reply:
yes, very much so!